meerkat_frontend/templates/includes/js.html

```diff
@@ -24,7 +24,7 @@
 <script src="{{ url_for('static', filename='js/bootstrap-datepicker.js') }}"></script>
 <script src="{{ url_for('static', filename='js/bootstrap-table.js') }}"></script>
 <script src="{{ url_for('static', filename='js/bootstrap-table-en-US.js') }}"></script>
-<script src="{{ url_for('static', filename='js/tableExport.min.js') }}"></script>
+<script src="{{ url_for('static', filename='js/tableExport.js') }}"></script>
```

Comment:
Shouldn't we use minified file for deployment?

Reply:
Yes we probably should, we need to write a mannual override for bower to do this though, I don't think the minified version is considered a main file (it was in a previous verion of the plugin which is why this change exists). I will make the change.